go/tools/builders/compilepkg.go

```diff
@@ -130,6 +132,10 @@ func compilePkg(args []string) error {
 		return fmt.Errorf("invalid test filter %q", testFilter)
 	}
 
+	if boringcrypto {
+		os.Setenv("GOEXPERIMENT", "boringcrypto")
```

Comment:
I don't like that we are relying on `os.Setenv` so much, but that's on us - no need to change anything.

Reply:
goit it